main/modules/files.js

```diff
@@ -420,6 +420,9 @@ function createRTFConversionFunction(sender) {
 }
 
 function createFromScrivener(importedPath, sender, isLoggedIntoPro) {
+  if (importedPath.error) {
```

Comment:
@jeanawhou, is this correct?  How can a string have an `error` property?

Reply:
I think its from the reducer 😅 

This is what i logged from the console when i import a non-scrivener/wrong format like folder
`importedPath { type: 'IMPORT_ERROR', error: 'Wrong file format' }`

Reply:
I think this is an error from the client before asking the server to import.
One sec while I confirm this suspicion.